var/spack/repos/builtin/packages/veloc/package.py

```diff
@@ -26,40 +27,29 @@ class Veloc(CMakePackage):
     version("1.1", sha256="2bbdacf3e0ce4e7c9e360874d8d85b405525bdc7bd992bdb1f1ba49218072160")
     version("1.0", sha256="d594b73d6549a61fce8e67b8984a17cebc3e766fc520ed1636ae3683cdde77cb")
 
-    # TODO: replace this with an explicit list of components of Boost,
-    # for instance depends_on('boost +filesystem')
-    # See https://github.com/spack/spack/pull/22303 for reference
-    depends_on(Boost.with_default_variants)
     depends_on("libpthread-stubs")
     depends_on("mpi")
+    depends_on("boost")
     depends_on("er")
     depends_on("axl@:0.3.0", when="@:1.5")
-    depends_on("openssl")  # Relies on the OpenSSL crypto library for checksums
-    depends_on("pdsh", when="@master")
-    depends_on("cmake@3.9:", type="build")
+    depends_on("axl@0.5.0:", when="@1.6:")
+    depends_on("openssl")
+    depends_on("cmake@3.10:", type="build")
 
-    with when("@1.6:"):
-        depends_on("axl@0.5.0:")
-        depends_on("redset")
-        depends_on("rankstr")
-        depends_on("shuffile")
-        depends_on("kvtree")
-
-    conflicts("%gcc@:4.9.3")
-
-    # requires C++11
     def flag_handler(self, name, flags):
         flags = list(flags)
         if name == "cxxflags":
-            flags.append(self.compiler.cxx11_flag)
+            if self.spec.satisfies("@:1.5"):
```

Comment:
1.6 was using cxx11 before. Is it safe to change this on packages depending on Veloc?

Reply:
This was wrong. 1.6 needs C++17. It probably went unnoticed because new compilers just issue warnings when features of the language introduced in higher standards are used (instead of errors).